docs/src/examples/components/Icon/Variations/IconExampleSpace.shorthand.tsx

```diff
@@ -4,29 +4,29 @@ import { Icon, Label } from '@stardust-ui/react'
 const IconExampleSpace = () => (
   <div>
     <p>
-      <Label content="Default" />
+      <Label>Default</Label>
```

Comment:
This was change because of console errors for invalid dom structure (div inside p element)

Reply:
Is <Label> referring to an HTML element or a Stardust element?

Reply:
It is Stardust component: https://stardust-ui.github.io/react/components/label